library/core/src/primitive_docs.rs

```diff
@@ -268,6 +268,47 @@ mod prim_bool {}
 /// [`Debug`]: fmt::Debug
 /// [`default()`]: Default::default
 ///
+/// # Never type fallback
+///
+/// When the compiler sees a value of type `!` it implicitly inserts a coercion (if possible),
```

Comment:
No comma required for a parenthetical aside:
```suggestion
/// When the compiler sees a value of type `!` it implicitly inserts a coercion (if possible)
```

Alternately, ditch the parens and use commas:
```suggestion
/// When the compiler sees a value of type `!` it implicitly inserts a coercion, if possible,
```